eva-server/src/test/java/uk/ac/ebi/eva/server/test/rule/FixSpringMongoDbRule.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.eva.server.test.rule;
+
+import com.lordofthejars.nosqlunit.mongodb.MongoDbConfiguration;
+import com.lordofthejars.nosqlunit.mongodb.SpringMongoDbRule;
+
+/**
+ * Temporary fix until nosql unit rc-6 or final is released
+ */
+public class FixSpringMongoDbRule extends SpringMongoDbRule {
```

Comment:
so I assume that you had to copy this class because we are not putting in eva-server a test dependency for eva-lib,  right? that could be done, but I'm ok duplicating this class if the test dependency is not straightforward.

Reply:
yes, I didn't bother setting up a test dependency just for the sake of this one class which might be removed anyway if we eventually upgrade to nosql-unit 1.0.